README.md

```diff
@@ -426,6 +445,57 @@ patchModules.config = [
 ]
 ``` 
 
+Compilation
+===
+
+Compilation to a specific Java release
+----
+
+For Java releases 6-8, see [Separate compilation of `module-info.java`](#separate-compilation-of-module-infojava).
```

Comment:
Add some more description *why* this is useful and when to use which option. You explained this really well in the issue, maybe move some of that to the docs.

Reply:
OK, will do.

Reply:
I added more description, although I didn't copy it from the issue, because it seemed to me I already copied all the necessary stuff to the "Separate compilation of `module-info.java`" section.

Please, let me know if the updated version reads well. I'll gladly reword/restructure it further if you provide me with any remarks you might have.